exe/kubernetes-deploy

```diff
@@ -10,6 +10,7 @@ template_dir = nil
 ARGV.options do |opts|
   opts.on("--skip-wait") { skip_wait = true }
   opts.on("--template-dir=DIR") { |v| template_dir = v }
+  opts.on("--timeout=TIMEOUT") { |v| KubernetesDeploy::KubernetesResource::OVERRIDDEN_TIMEOUT = v.to_i }
```

Comment:
Yes you want to store it in a class attribute, like `logger` in `KubernetesResource`.